### DISCUSSION TITLE
Welcome to Microsoft Graph PowerShell Discussions!

### DISCUSSION THREAD

**Discussion by peombwa** [microsoftgraph/msgraph-sdk-powershell#1699] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  Discussions on GitHub is an experiment for the Microsoft Graph PowerShell team, so we'll continue to evaluate how to use this feature (or not use it) pending community feedback.

**Comment by i5513** [comment]
Seems like no body is reading (or at least replying discussions), is better use issues? where can we talk/ask about MS Graph?
